pkg/grpc/grpc.go

```diff
@@ -19,6 +19,10 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+const (
+	grpcServiceConfig = `{"loadBalancingPolicy":"round_robin"}`
```

Comment:
Can you please add short comment here? why we need this ?

Reply:
When you say this, do you mean the const?

Reply:
Added

Reply:
> When you say this, do you mean the const?

yes, it would be good to explain about this option for the other contributors...